lib/api/v3/work_packages/form/work_package_payload_representer.rb

```diff
@@ -64,6 +64,13 @@ def initialize(represented, options = {})
 
           property :lock_version
           property :subject, render_nil: true
+          property :done_ratio,
+                   as: :percentageDone,
+                   getter: -> (*) { done_ratio if Setting.work_package_done_ratio != 'disabled' },
+                   render_nil: false
+          property :estimated_hours,
```

Comment:
I talked with @ulferts about this and the decision was made to push this back for now in order to get this done.